src/views/menus/useGlobalCommands.tsx

```diff
@@ -154,5 +141,27 @@ export const useGlobalCommands = (): MenuConfig<string, string> => {
         onSelect: () => navigate(`${AppRoute.Trade}/${market}`),
       })),
     },
+    {
+      // TODO: Remove this group when available under display settings
+      group: 'layout',
+      groupLabel: 'Layout', // TODO: i18n
+      items: [
+        {
+          value: LayoutItems.setDefaultLayout,
+          label: 'Set Default Layout',
```

Comment:
nit, I think these keys exist [here](https://github.com/dydxprotocol/v4-localization/blob/main/config/localization/en/app.json#L2007C8-L2007C26)! (though layout is still missing)